pkg/utils/machines/machines.go

```diff
@@ -4,49 +4,19 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/golang/glog"
 	mrv1 "kubevirt.io/machine-remediation-operator/pkg/apis/machineremediation/v1alpha1"
-	"kubevirt.io/machine-remediation-operator/pkg/utils/conditions"
+
+	"github.com/golang/glog"
 
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
 
 	mapiv1 "sigs.k8s.io/cluster-api/pkg/apis/machine/v1beta1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// IsMachineHealthy returns true if the the machine is running and machine node is healthy
```

Comment:
hm why do you move it?

Reply:
To avoid circular dependency between packages `conditions` and `machines` as this function is only used in MHC controller. I added some functions to `conditions` pkg which require functions from `machine` pkg. To avoid all of this, there will be some more refactoring required.